src/main/java/Cleese/Task.java

```diff
@@ -0,0 +1,32 @@
+package Cleese;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
```

Comment:
Might want to change this to private instead of protected. For java, unlike C++ where protected makes the variable inaccessible to other classes besides the derived classes, in java, as long as a class is in the same package, they can access the protected variable.

If that is not your intention, you may want to change to private.